vendor/vendor.json

```diff
@@ -1,6 +1,6 @@
 {
 	"comment": "",
-	"ignore": "test appengine",
```

Comment:
was this intentional?

Reply:
I wouldn't have thought so. @harveylowndes can you remove it please?